pkg/gofr/external_db.go

```diff
@@ -183,3 +183,14 @@ func (a *App) AddSurrealDB(db container.SurrealBDProvider) {
 	db.Connect()
 	a.container.SurrealDB = db
 }
+
+func (a *App) AddElasticSearch(db container.ElasticsearchProvider) {
```

Comment:
I found one last one, maybe the documentation needs to be updated

```suggestion
func (a *App) AddElasticsearch(db container.ElasticsearchProvider) {
```

Reply:
changed here too and i checked in the documenation it is updated.